### PR TITLE
Add IsMobileDevice for HttpBrowserCapabilities.

### PR DESCRIPTION
**Pull request by johnLwith**
This PR to fix issue: https://github.com/dotnet/systemweb-adapters/issues/222.
@twsouthwick  @adityamandaleeka

### CODE REVIEW

**Comment by adityamandaleeka** [comment]
@johnLwith Thanks for the PR. It looks like there's an issue with type forwarding setup. @twsouthwick should hopefully be able to help sort that out in a week or so.

**Comment by twsouthwick** [comment]
@johnLwith can you run `./update_apis.ps1` in the solution directory? It will update the autogenerated stuff.

**Comment by johnLwith** [comment]
@twsouthwick Thanks for you helping.  It looks work.

File: test/Microsoft.AspNetCore.SystemWebAdapters.Tests/HttpBrowserCapabilitiesTests.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.Tests
+{
+    public class HttpBrowserCapabilitiesTests
+    {
+        [Theory]
+        [InlineData("", false)]
+        [InlineData("Mozilla/5.0 (iPhone; CPU iPhone OS 12_0_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0 Mobile/15E148 Safari/604.1", true)]
+        [InlineData("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36", false)]
+        public void IsMobileDevice(string userAgent, bool expected)
```
Comment (twsouthwick) [inline test/Microsoft.AspNetCore.SystemWebAdapters.Tests/HttpBrowserCapabilitiesTests.cs:14]: There are already tests here: BrowserCapabilitiesFactoryTests. Not as discoverable, but could you put these there?

Comment (johnLwith) [reply test/Microsoft.AspNetCore.SystemWebAdapters.Tests/HttpBrowserCapabilitiesTests.cs]: Updated.

**Review by twsouthwick — APPROVED** [review]
LGTM. Thanks @johnLwith

**Pull request merged**